docs/components/search/index.css

```diff
@@ -24,6 +24,12 @@
   left: calc(50% - 350px);
 }
 
+.searchBlock {
+  color: #6c6c6c;
+  font-weight: normal;
+  text-transform: none;
```

Comment:
Do these styles do anything?

Reply:
I say this because I suspect that they don't, what happens if you remove them?

Reply:
Yeah the default button font is tiny because of our custom buttons, this definitely does do something if I remove.